recorder/src/main/java/com/mapbox/vision/recorder/RecordingActivity.kt

```diff
@@ -0,0 +1,119 @@
+package com.mapbox.vision.recorder
+
+import android.annotation.SuppressLint
+import android.os.Environment
+import android.view.View
+import com.mapbox.vision.VisionManager
+import com.mapbox.vision.common.BaseActivity
+import com.mapbox.vision.common.view.hide
+import com.mapbox.vision.common.view.show
+import com.mapbox.vision.mobile.core.interfaces.VisionEventsListener
+import com.mapbox.vision.mobile.core.models.Camera
+import com.mapbox.vision.mobile.core.models.Country
+import com.mapbox.vision.performance.ModelPerformance
+import com.mapbox.vision.performance.ModelPerformanceConfig
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+import kotlinx.android.synthetic.main.activity_recording.*
+
+class RecordingActivity : BaseActivity() {
+
+    companion object {
+        private val BASE_SESSION_PATH = "${Environment.getExternalStorageDirectory().absolutePath}/Telemetry"
```

Comment:
Now that I think about name it doesn't seem like an understandable one. Seems smth with `Vision` will work much better, `VisionTelemetry`/`MapboxVisionTelemetry` etc. ?

Reply:
done